google-cloud-datastore/src/main/java/com/google/cloud/datastore/TransactionImpl.java

```diff
@@ -16,6 +16,8 @@
 
 package com.google.cloud.datastore;
 
+import static com.google.cloud.datastore.ReadOption.transactionId;
```

Comment:
I would be inclined to revert this, as it makes the readability more confusing (`transactionId(transactionId)`)

Reply:
how about we change the method name to `withTransaction`? In that case it becomes `withTransaction(transactionId)`.

Reply:
that sounds good!

Reply:
or `setTransactionId` also sgtm

Reply:
addressed in https://github.com/googleapis/java-datastore/pull/1041.